src/components/tou-modal/variations/tou-under-16.jsx

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable react/jsx-no-bind */
+import React, {useState} from 'react';
+import Modal from '../../modal/base/modal.jsx';
+import {FormattedMessage, injectIntl} from 'react-intl';
+import {TermsOfUseLink} from '../tou-modal.jsx';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line arrow-body-style
+const TosModalUnder16 = ({email, isOpen, onClose}) => {
+    const [currentStep, setCurrentStep] = useState(1);
+    const handleNextStep = () => {
+        setCurrentStep(prevStep => prevStep + 1);
+    };
+
+    return (
+        <Modal
+            overlayClassName="tou-modal-overlay"
+            className="tou-modal"
+            showCloseButton={currentStep === 2}
+            isOpen={isOpen}
+            onRequestClose={() => {
+                if (currentStep === 2) {
+                    onClose();
+                }
+            }}
+        >
+            <div className="tou-modal-top" />
+            <div className="tou-modal-content">
+                {currentStep === 1 ? (
+                    <Step1
+                        onNextStep={handleNextStep}
+                        email={email}
+                    />
+                ) : (
+                    <TosEmailSentStep />
+                )}
```

Comment:
I am wondering and this is not really a must but do you think it would be better to have arrays with the steps and to display the component on the index corresponding to the currentStep?